<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>T3_AF_faceHistory</class>
 <widget class="QDialog" name="T3_AF_faceHistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="_exitPushBtn_">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>70</y>
     <width>55</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;border-width:0;border-style:outset;color:white</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="_backgroungLabel_">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>405</width>
     <height>360</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:
url(:/Pictures/faceHistory_background.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="_faceInfoTableView_">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>97</y>
     <width>351</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;border-width:0;border-style:outset;color:white</string>
   </property>
  </widget>
  <widget class="QLabel" name="_wholeBack_">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/Pictures/confirmExit_background.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>_wholeBack_</zorder>
  <zorder>_backgroungLabel_</zorder>
  <zorder>_exitPushBtn_</zorder>
  <zorder>_faceInfoTableView_</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
